psutil/tests/__init__.py

```diff
@@ -130,6 +131,7 @@
 CI_TESTING = APPVEYOR or GITHUB_ACTIONS
 # are we a 64 bit process?
 IS_64BIT = sys.maxsize > 2 ** 32
+MACOS_ARM64 = MACOS and platform.machine() == 'arm64'
```

Comment:
I would define this as a generic `ARM64` instead of making it specific for macOS.
Regardless, since it's only being used once across the entire code base, I think you can simply use `platform.machine()` later, in the test itself.